enterprise/frontend/src/embedding/data-picker/DataSelector.jsx

```diff
@@ -672,22 +571,13 @@ export class UnconnectedDataSelector extends Component {
     const loadersForSteps = {
       // NOTE: make sure to return the action's resulting promise
       [DATA_BUCKET_STEP]: () => {
-        return Promise.all([
-          this.props.fetchDatabases(this.props.databaseQuery),
-          this.props.fetchDatabases({ saved: true }),
-        ]);
+        return this.props.fetchDatabases();
```

Comment:
since every `fetchDatabases` call passes `{ saved: true }` it would make more sense to do that inside the function itself.